### DISCUSSION TITLE
SyntaxError: \node_modules\react-table\src\publicUtils.js: Unexpected token (210:34)

### DISCUSSION THREAD

**Discussion by hayk94** [TanStack/table#2270] [General]
Hey guys, I am getting this error, can't figure out why

Any ideas?

```
./node_modules/react-table/src/publicUtils.js
SyntaxError: \node_modules\react-table\src\publicUtils.js: Unexpected token (210:34)

  208 | 
  209 | export function flexRender(Comp, props) {
> 210 |   return isReactComponent(Comp) ? <Comp {...props} /> : Comp
      |                                   ^
  211 | }
```

**Comment by mykola-blu** [comment]
The same situation

**Reply by hayk94** [reply]
Hey man, this was back a while so I don’t remember exactly what was the fix.
But as much as memory serves it had something to do with returning nothing - `undefined` from some components during some conditions and returning `null` fixed it.

**Reply by mykola-blu** [reply]
Thanks man! You mean you change it in your webpack.config or, in publicUtils.js?

**Reply by hayk94** [reply]
In the components that I was passing to react table’s `Cell` properties
